figures/S3.supplemental_figure3/scripts/S3.r

```diff
@@ -0,0 +1,1060 @@
+            'Flagellin 1.0 ug/ml - Disulfiram 1.0 uM, Pyroptosis',
+            'LPS 0.01 ug/ml - DMSO 0.025%, Pyroptosis',
+            'LPS 0.1 ug/ml - DMSO 0.025%, Pyroptosis',
+            'Flagellin 0.1 ug/ml - DMSO 0.0%, NA',
+            'Flagellin 0.1 ug/ml - DMSO 0.025%, Control',
+            'Disulfiram 0.1 uM - DMSO 0.025%, Control',
+            'LPS 1.0 ug/ml + Nigericin 1.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 1.0 ug/ml + Nigericin 10.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 1.0 ug/ml + Nigericin 10.0 uM - Disulfiram 1.0 uM, Pyroptosis',
+            'LPS 1.0 ug/ml + Nigericin 10.0 uM - Z-VAD-FMK 100.0 uM, Control',
+            'LPS 1.0 ug/ml + Nigericin 3.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 1.0 ug/ml - DMSO 0.025%, Pyroptosis',
+            'Disulfiram 1.0 uM - DMSO 0.025%, Control',
+            'Thapsigargin 1.0 uM - DMSO 0.025%, Apoptosis',
+            'Topotecan 10.0 nM - DMSO 0.025%, Control',
+            'LPS 10.0 ug/ml - DMSO 0.025%, Pyroptosis',
+            'LPS 10.0 ug/ml - Disulfiram 0.1 uM, Pyroptosis',
+            'LPS 10.0 ug/ml - Disulfiram 1.0 uM, Pyroptosis',
+            'LPS 10.0 ug/ml - Disulfiram 2.5 uM, Pyroptosis',
+            'LPS 10.0 ug/ml - Z-VAD-FMK 100.0 uM, Control',
+            'Thapsigargin 10.0 uM - DMSO 0.025%, Apoptosis',
+            'H2O2 100.0 nM - DMSO 0.025%, Control',
+            'LPS 100.0 ug/ml + Nigericin 1.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 100.0 ug/ml + Nigericin 10.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 100.0 ug/ml + Nigericin 3.0 uM - DMSO 0.025%, Pyroptosis',
+            'LPS 100.0 ug/ml - DMSO 0.025%, Pyroptosis',
+            'H2O2 100.0 uM - DMSO 0.025%, Control',
+            'H2O2 100.0 uM - Disulfiram 1.0 uM, Control',
+            'H2O2 100.0 uM - Z-VAD-FMK 100.0 uM, Control',
+            'Disulfiram 2.5 uM - DMSO 0.025%, Control',
+            'Topotecan 20.0 nM - DMSO 0.025%, Control',
+            'Topotecan 5.0 nM - DMSO 0.025%, Control',
+            'Media ctr 0.0 0, Control'
+        ),
+        values = c(
+            18,
+            18,
+            18,
+            18,
+            16,
+            16,
+            16,
+            16,
+            18,
+            18,
+            18,
+            16,
+            16,
+            16,
+            18,
+            16,
+            16,
+            18,
+            17,
+            18,
+            16,
+            16,
+            16,
+            16,
+            18,
+            17,
+            18,
+            16,
+            16,
+            16,
+            16,
+            18,
+            18,
+            18,
+            18,
+            18,
+            18,
+            18
+        )
+    )
+    + theme(legend.position = "bottom")
+    # set the legend columns to 4
+    + guides(color = guide_legend(ncol = 2))
+
+
+)
+
+# options(repr.plot.width=10, repr.plot.height=5)
```

Comment:
Recommend removing commented out code if not needed.

Reply:
removed